dev/.scalafmt.conf

```diff
@@ -22,6 +22,13 @@ align.tokens = []
 optIn = {
   configStyleArguments = false
 }
-danglingParentheses = false
-docstrings = JavaDoc
+danglingParentheses.preset = false
+docstrings.style = Asterisk
 maxColumn = 98
+runner.dialect = scala212
+fileOverride {
+  "glob:**/src/**/scala-2.13/**.scala" {
+    runner.dialect = scala213
+  }
+}
+version = 3.5.9
```

Comment:
where is this version from? is it from Maven plugin's dependency?

Reply:
### why version 3.5.9 ?
Buildin version(Maven plugin's dependency) is : 3.2.1
But current max version is : 3.5.9 (https://mvnrepository.com/artifact/org.scalameta/scalafmt-core)
<img width="890" alt="image" src="https://user-images.githubusercontent.com/15246973/187428866-35b7aa1e-f7b1-41b2-9222-a1e87d85aca7.png">

#### If set version = 3.5.9, plugin will dynamically download as following:
code: https://github.com/scalameta/scalafmt/blob/master/scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtModuleLoader.scala#L25-L39
<img width="724" alt="image" src="https://user-images.githubusercontent.com/15246973/187427653-f7feea71-4358-4ff7-96d3-5da9d3beee64.png">

### root cause
<img width="797" alt="image" src="https://user-images.githubusercontent.com/15246973/187427474-a2fe19e8-d6a4-4edb-bebc-5ea2b068f0b2.png">